docs/index.md

```diff
@@ -50,155 +55,91 @@ Compose has commands for managing the whole lifecycle of your application:
-
-### Build and run your app with Compose
-
-Now, when you run `docker-compose up`, Compose will pull a Redis image, build an image for your code, and start everything up:
-
-    $ docker-compose up
-    Pulling image redis...
-    Building web...
-    Starting composetest_redis_1...
-    Starting composetest_web_1...
-    redis_1 | [8] 02 Jan 18:43:35.576 # Server started, Redis version 2.8.3
-    web_1   |  * Running on http://0.0.0.0:5000/
-    web_1   |  * Restarting with stat
-
-If you're using [Docker Machine](https://docs.docker.com/machine), then `docker-machine ip MACHINE_VM` will tell you its address and you can open `http://MACHINE_VM_IP:5000` in a browser.
-
-If you're using Docker on Linux natively, then the web app should now be listening on port 5000 on your Docker daemon host. If http://0.0.0.0:5000 doesn't resolve, you can also try http://localhost:5000.
-
-You should get a message in your browser saying:
-
-`Hello World! I have been seen 1 times.`
-
-Refreshing the page will increment the number.
-
-If you want to run your services in the background, you can pass the `-d` flag
-(for "detached" mode) to `docker-compose up` and use `docker-compose ps` to
-see what is currently running:
+An important part of any Continuous Deployment or Continuous Integration process
+is the automated test suite. Automated end-to-end testing requires an
+environment in which to run tests. Compose provides a convenient way to create
+and destroy isolated testing environments for your test suite. By defining the full
+environment in a [Compose file](yml.md) you can create and destroy these
+environments in just a few commands:
 
     $ docker-compose up -d
-    Starting composetest_redis_1...
-    Starting composetest_web_1...
-    $ docker-compose ps
-	    Name                 Command            State       Ports
-    -------------------------------------------------------------------
-    composetest_redis_1   /usr/local/bin/run         Up
-    composetest_web_1     /bin/sh -c python app.py   Up      5000->5000/tcp
-
-The `docker-compose run` command allows you to run one-off commands for your
-services. For example, to see what environment variables are available to the
-`web` service:
-
-    $ docker-compose run web env
-
-See `docker-compose --help` to see other available commands. You can also install [command completion](completion.md) for the bash and zsh shell, which will also show you available commands.
+    $ ./run_tests
+    $ docker-compose stop
+    $ docker-compose rm -f
 
-If you started Compose with `docker-compose up -d`, you'll probably want to stop
-your services once you've finished with them:
+### Single host deployments
 
-    $ docker-compose stop
+Compose has traditionally been focused on development and testing workflows,
+but with each release we're making progress on more production-oriented features.
+Compose can be used to deploy to a remote docker engine, for example a cloud
```

Comment:
This sentence is awkward seems a bit packed with ideas.  Let me see if I can clarify it by unpacking it a bit. (adjust if I misunderstood what you were getting at.) Also, three links in the space of two sentences is just noisy --- like putting everything in bold. If they want to get to Machine/Swarm they can get there easily enough --- I'd prefer to focus on compose.

--->

Beyond development and testing workflows, you can use Compose for production-oriented features. For example, you can use Compose to deploy to a remote Docker Engine instance. This instance may be a single instance provisioned by Docker Machine or an entire cluster built through Docker Swarm.

For details on using production-oriented features, see  [compose in production](production.md) in this documentation.